lib/routes/scroll/utils.ts

```diff
@@ -0,0 +1,90 @@
+
+// Helper function to map article data to a consistent format
+function mapArticle(article: any) {
+    return {
+        title: article.title,
+        link: article.permalink,
+        description: article.summary || '',
+        itunes_item_image: article.cover?.large || '',
+        image: article.cover?.large || '',
+        author:
+            article.authors?.length > 0
+                ? article.authors.map((author: any) => ({
+                      name: author?.name,
+                      url: `${rootUrl}/author/${author?.id}`,
+                      avatar: author?.headshot || '',
+                  }))
+                : '',
+        pubDate: article.published ? parseDate(article.published) : undefined,
+    };
+}
+
+export async function extractFeedArticleLinks(feedPath: string) {
+    const feedUrl = `${rootUrl}/feed/${feedPath}/page/1`;
+    const response = await ofetch(feedUrl);
+
+    if (!response.articles) {
+        return [];
+    }
+
+    const allArticles = response.articles.flatMap((section: any) => section.blocks?.filter((block: any) => block.type === 'row-stories').flatMap((block: any) => block.articles || []) || []);
+
+    return allArticles.map((article) => mapArticle(article)).filter((item: any) => item.title && item.link);
+}
+
+export async function extractTrendingArticles() {
+    const feedUrl = `${rootUrl}/feed/series/1/page/1`;
+    const response = await ofetch(feedUrl);
+
+    if (!response.articles) {
+        return [];
+    }
+
+    // Filter for trending blocks which have title: "Trending"
+    const trendingBlocks = response.articles.flatMap((section: any) => section.blocks || []).filter((block: any) => block.title === 'Trending' && block.type === 'list-collection-stories');
+
+    if (!trendingBlocks.length) {
+        return [];
+    }
+
+    // Combine articles from all trending blocks
+    const trendingArticles = trendingBlocks.flatMap((block: any) => block.articles || []);
+
+    return trendingArticles.map((article) => mapArticle(article)).filter((item: any) => item.title && item.link);
+}
+
+export function fetchArticleContent(item: any): Promise<DataItem> {
+    return cache.tryGet(item.link, async () => {
+        try {
+            const articleResponse = await ofetch(item.link);
+            const $article = load(articleResponse);
+
+            // Remove elements that shouldn't be in the content
+            $article('.below-article-share-block').nextAll().remove();
+            $article('.below-article-share-block').remove();
+            $article('header').remove(); // Remove all header tags
+
+            // Get content after header removal
+            const content = $article('.story-element').html() || $article('article .content').html() || $article('.article-content').html() || item.description;
+
+            const tags = $article('.article-tags-list a.tag-menu')
+                .toArray()
+                .map((tag) => $article(tag).text().trim());
```

Comment:
It's still https://github.com/DIYgod/RSSHub/pull/19564#discussion_r2195332959. Extract them to `category` and remove them from the main article. This also includes the `We welcome your comments at...`